app/models/access_token.rb

```diff
@@ -38,7 +38,7 @@ def public?
     private
 
     def non_public_scopes
-      %w(cms).freeze
+      %w[].freeze
```

Comment:
I think we should remove this method as it's just an empty Array now.

Reply:
I'd prefer to leave it empty, so we keep the structure to add future items to it if needed. If we remove it, then the `public?` method above would make no sense neither and should be removed too, also the `visible_for` method should be refactored. Is it worth it?

Reply:
if we don't have any private APIs, then why have these?

Reply:
> Are we sure we won't have any in the future?

I'm pretty sure no one is really sure 😆
If we're gonna leave this structure here, I'd ask for a comment in this method saying something like "This method exists because we need the flexibility to have private APIs. As we have none now, it returns an empty string".

Reply:
Why should we have private APIs?

Reply:
I removed the support for the private APIs